front/pages/api/w/[wId]/data_sources/[name]/managed/update.ts

```diff
@@ -41,6 +39,9 @@ async function handler(
     });
   }
 
+  const owner = auth.getNonNullableWorkspace();
```

Comment:
cf throw error VS return 400s and checking that callers behaves the same for the two

Reply:
will stop marking those from here on

Reply:
same :)